src/librustc_codegen_ssa/lib.rs

```diff
@@ -122,6 +123,14 @@ pub enum ModuleKind {
     Allocator,
 }
 
+#[derive(Copy, Clone, Debug, PartialEq)]
+pub enum CallKind<'tcx> {
```

Comment:
Document this type. It should be clear both here and at the use sites that this is solely for debug info and does not actually affect codegen